### ISSUE TITLE
terraform-docs M1 support?

### ISSUE THREAD

**Issue by fperk** [terraform-docs/terraform-docs#379]
Hi all,

when i try to install terraform-docs via brew i get the following message:

```
brew install --build-from-source terraform-docs
==> Downloading https://github.com/terraform-docs/terraform-docs/archive/v0.10.1
Already downloaded: /Library/Caches/Homebrew/downloads/82444f14d8987cbd48a6c37fb5e5b410776ea11ffb619029d5da57f5b66eb765--terraform-docs-0.10.1.tar.gz
==> make build
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - bin/darwin-amd64/terraform-docs
```

Is there a workaround to get terraform-docs installed and is there a plan to support M1?

**Comment by mbarrien** [comment]
This just got merged a few hours ago: https://github.com/Homebrew/homebrew-core/pull/70484

**Comment by khos2ow** [comment]
Thanks @mbarrien for updating the formula, but it seems that won't be enough. Up to go1.16 (which is not officially released yet) `darwin/arm64` refers to ios and not macos. https://github.com/golang/go/issues/38485 has more information.

**Comment by mbarrien** [comment]
I can say that the Homebrew formula does work for me, both the one I built locally, and the one that they built once they rebuilt the Homebrew bottle.

**Comment by khos2ow** [comment]
Have you built and tested it on new Apple Silicon? Can it be cross-compiled on another machine?

**Comment by mbarrien** [comment]
Yes, this was compiled and tested on Apple Silicon Mac M1 (I did the port because I'm testing the M1's for my engineering org.) Note that this test was just being done via the Homebrew build (which builds from source), not via the goreleaser.

**Comment by khos2ow** [comment]
And I assume you're using go1.1, right? Anyways that's a good news because eventually goreleaser will catch up, but that means for some time (if any) we won't have a binary dedicated for Apple Silicon directly downloadable from GitHub release page and only homebrew would work, which is fine considering all the moving parts.

**Comment by fperk** [comment]
Thanks, the installation works now even without the `--build-from-source`, just with `brew install terraform-docs`.

I have the `Homebrew 3.0.0` and `go version go1.16rc1 darwin/arm64`

**Comment by khos2ow** [comment]
We've released `v0.11.0` and the corresponding brew formula has also been updated, can you verify that this works as expected on M1?

Also I'd like to still keep this issue open until go1.16 is finally released and we could officially support it for all the available release assets (i.e. currently there's no `darwin/arm64` in github release assets)

**Comment by skyzyx** [comment]
**Status update:** Go 1.16 was released today with [non-prerelease, first-class support for `darwin/arm64`](https://golang.org/doc/go1.16#darwin).

**Comment by khos2ow** [comment]
That's great @skyzyx, I assume by the time we'll be ready to cut a new release GoReleaser will have adopted go1.16 too.

**Comment by fperk** [comment]
> We've released `v0.11.0` and the corresponding brew formula has also been updated, can you verify that this works as expected on M1?
> 
> Also I'd like to still keep this issue open until go1.16 is finally released and we could officially support it for all the available release assets (i.e. currently there's no `darwin/arm64` in github release assets)

I tested it again on Big Sur 11.2.1, M1 MB Air and it worked without problems.

The installed version is : `terraform-docs version v0.11.1 darwin/arm64 BuildDate: 2021-02-18T10:44:22-0800`
